src/rules/noUnusedVariableRule.ts

```diff
@@ -95,7 +95,8 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
 
     // skip exported variables
     public visitVariableStatement(node: ts.VariableStatement): void {
-        if (this.hasModifier(node.modifiers, ts.SyntaxKind.ExportKeyword)) {
+        if (this.hasModifier(node.modifiers, ts.SyntaxKind.ExportKeyword) ||
+            this.hasModifier(node.modifiers, ts.SyntaxKind.DeclareKeyword)) {
```

Comment:
please put the boolean operator on the next line for consistency.
yes, we ought to have a tslint rule that checks this :)